docker/image/whyis-jetty/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM jetty:9.2-jre8
+
+USER root
+RUN apt-get update && apt-get install -y unzip
+
+# With the blazegraph/ context
+COPY ./resources/blazegraph.war /tmp/.
+
+# Creates the directory
+# WORKDIR /var/lib/jetty/webapps/blazegraph
+WORKDIR /usr/local/jetty/webapps/blazegraph
+
+RUN unzip -u /tmp/blazegraph.war
```

Comment:
Can we put this in a script, copy the script into the image, and invoke it? Having this many RUNs is pretty gnarly.